go/vt/vttablet/onlineddl/schema.go

```diff
@@ -77,6 +77,8 @@ const (
 	alterSchemaMigrationsTableVreplLivenessIndicator   = "ALTER TABLE _vt.schema_migrations add column vitess_liveness_indicator bigint NOT NULL DEFAULT 0"
 	alterSchemaMigrationsTableUserThrottleRatio        = "ALTER TABLE _vt.schema_migrations add column user_throttle_ratio float NOT NULL DEFAULT 0"
 	alterSchemaMigrationsTableSpecialPlan              = "ALTER TABLE _vt.schema_migrations add column special_plan text NOT NULL"
+	alterSchemaMigrationsLastThrottled                 = "ALTER TABLE _vt.schema_migrations add column last_throttled_timestamp timestamp NULL DEFAULT NULL"
+	alterSchemaMigrationsComponentThrottled            = "ALTER TABLE _vt.schema_migrations add column component_throttled tinytext NOT NULL"
```

Comment:
We're running out on `varchar` capacity in `_vt.schema_migrations`, which is why I now use `*text` types for everything. But, I'm curious, isn't `tinytext` an equivalent of `varchar(255)`? And it can be indexed just as with a `varchar`? At least I've always indexed `*text` types without a problem.